nucypher/characters/chaotic.py

```diff
@@ -174,55 +167,54 @@ def gather_encrypted_decryption_shares(
         # We only really need one encrypted tdr.
         etdr = list(encrypted_requests.values())[0]
 
+        # decrypt request
+        threshold_decryption_request = trdp.decrypt_encrypted_request(etdr)
+        ciphertext = threshold_decryption_request.ciphertext
+        conditions = str(threshold_decryption_request.conditions).encode()
+        ritual_id = threshold_decryption_request.ritual_id
+        variant = threshold_decryption_request.variant
+
+        # We can obtain the transcripts from the side-channel (deserialize) and aggregate them
+        validator_messages = [
+            ferveo.ValidatorMessage(validator, transcript)
+            for validator, transcript in self._learner._dkg_insight.transcripts
+        ]
+        aggregate = ferveo.AggregatedTranscript(validator_messages)
+        assert aggregate.verify(
+            self._learner._dkg_insight.shares_num,
+            # TODO this list should have to be passed again (either make `verify` static or use list
+            #  provided in constructor
```

Comment:
Possibly material for `ferveo` issue

Reply:
Indeed. This list is passed in the constructor, and then again in the `verify` function - which is redundant.